opensourceleg/actuators/moteus.py

```diff
@@ -278,68 +449,77 @@ def start(self):
                 servo_bus_map = self.bus_map
             )
         
-    def update(self):
-        
-        # self._data = await self.transport.cycle(
-        #     self._commands
-        # )
-        pass
-        
+    async def update(self):
+        # TODO: multiple servo syncing process should go here
```

Comment:
Can you please add these TODOs as feature requests on the issue page and link them to the project board so that people can pick this up in the future?